src/widgetastic_patternfly4/duallistselector.py

```diff
@@ -0,0 +1,132 @@
+from widgetastic.widget import GenericLocatorWidget
+from widgetastic.widget import TextInput
+
+from .button import Button
+
+
+class BaseDualListSelector:
+    """Represents the Patternfly-4 Dual list selector.
+
+    https://www.patternfly.org/v4/components/dual-list-selector
+    """
+
+    LIST_ITEMS = ".//div[contains(@class, 'dual-list-selector__menu')]"
+    SECTION_TITLE = ".//div[@class='pf-c-dual-list-selector__title-text']"
+    ITEMS = ".//li[contains(@class, 'dual-list-selector__list-item')]"
+    SELECTED_ITEMS = (
+        ".//li[@aria-selected='true' and contains(@class, 'dual-list-selector__list-item')]"
+    )
+    move_all_right = Button(locator=".//button[@aria-label='Add all']")
+    move_selected_right = Button(locator=".//button[@aria-label='Add selected']")
+    move_selected_left = Button(locator=".//button[@aria-label='Remove selected']")
+    move_all_left = Button(locator=".//button[@aria-label='Remove all']")
+
+    @property
+    def _left_list(self):
+        return self.browser.elements(self.LIST_ITEMS, parent=self)[0]
```

Comment:
I think we should try to avoid indexing.
can we create locators like this 
`.//div[contains(@class, 'pf-m-available')]//div[contains(@class, 'dual-list-selector__menu')]`

Same for chosen.

Reply:
Agree, great catch! fixed